project-novica/demo/src/main/resources/static/javascript.js

```diff
@@ -0,0 +1,18 @@
+function logIn() {
+    var UserDTO = {};
```

Comment:
Rename variable name to userDTO and rename the file to some proper name.
Example: loginUser.js